app/views/users/index.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1>Users List</h1>
-<p>Here is a list of all usres</p>
+<section class="background">
+    <h1 class="users-title">Welcome to Users page</h1>
+    <ul>
+      <% @users.each do |user|%>
+        <%= link_to user do %>
+          <li class="card flex hover">
+            <div class="user-image border">
+              <%= image_tag('R.png',  :width => "100px", :heigth=>"100px") %>
+            </div>
+            <div class="user-data border">
+              <h1><%= user.name %></h1>
+              <p class="number-posts">Number of posts: <%= user.post_counter %></p>
+            </div>
+          </li>
+        <% end %>
+      <% end %>
```

Comment:
- [ ] **[OPTIONAL]** Usually, it is good practice to move iterations like this in views to partials. It also helps keep your code clean. I would suggest you do the same here.

  ## SUGGESTION
  You can create a `app/views/users/_users.html.erb` file and export this highlighted code to the file and then render the partial in this file like so:
  ```
  <section class="background">
    <h1 class="users-title">Welcome to Users page</h1>
    <ul>
      <%= render 'users' %>
    </ul>
  </section>
  ```

Reply:
@Tobinchilongo yeah. I marked it optional. It's not required.

Reply:
Hi @TedLivist 

I have sent this <PR> for a review already